docs/reference/kusk-cli.md

```diff
@@ -0,0 +1,206 @@
+3. Move it to a location in the PATH. For example `mv kusk_0.1.0_Linux_arm64/kusk /usr/local/bin/kusk`
+
+For Windows, download the binary from [here](https://github.com/kubeshop/kusk/releases), unpack the binary and add it to `%PATH%`. 
+
+# Updating
+
+[(Back to top)](#table-of-contents)
+
+## Homebrew
+`brew upgrade kubeshop/kusk/kusk`
+
+## Latest release on Github
+`go install github.com/kubeshop/kusk@$VERSION`
+
+## From source
+Insde of the kusk repository directory
+```
+git pull && go install
+```
+
+# Commands
+
+[(Back to top)](#table-of-contents)
+
+## Install
+The install command will install Kusk Gateway and all its components with a single command. Kusk uses Helm to do this so you will need to have [Helm installed](https://helm.sh/docs/intro/install/).
+
+Kusk Gateway Components:
+* **Kusk Gateway manager** - responsible for updating and rolling out the envoy configuration to your envoy fleets as your deploy APIs and Static Routes.
+* **Envoy Fleet** - responsible for exposing and routing to your APIS and frontends
+* **Kusk Gateway API** - REST API which is exposed by Kusk Gateway and allows you to programatically query which APIs, Static Routes and Envoy Fleets are deployed.
+* **Kusk Gateway Dashboard** - a web UI for Kusk Gateway where you can deploy APIS and see which APIs, StaticRoutes and EnvoyFleets are deployed.
+
+### Flags
+|         Flag         |                                                     Description                                                     | Required? |
+|:--------------------:|:-------------------------------------------------------------------------------------------------------------------:|:---------:|
+|       `--name`       | the prefix of the name to give to the helm releases for each of the kusk gateway components (default: kusk-gateway) |     ❌     |
+| `--namespace` / `-n` |   the namespace to install kusk gateway into. Will create the namespace if it doesn't exist (default: kusk-system)  |     ❌     |
+|   `--no-dashboard`   |                               when set, will not install the kusk gateway dashboard.                                |     ❌     |
+|      `--no-api`      |                       when set, will not install the kusk gateway api. implies --no-dashboard.                      |     ❌     |
+|  `--no-envoy-fleet`  |                                     when set, will not install any envoy fleets                                     |     ❌     |
+
+### Examples
+```
+$ kusk install
+
+Will install kusk-gateway, a public (for your APIS) and private (for the kusk dashboard and api) 
+envoy-fleet, api, and dashboard in the kusk-system namespace using helm.
+
+$ kusk install --name=my-release --namespace=my-namespace
+
+Will create a helm release named with --name in the namespace specified by --namespace.
+
+$ kusk install --no-dashboard --no-api --no-envoy-fleet
+
+Will install kusk-gateway, but not the dashboard, api, or envoy-fleet.
+```
+
+## Api generate
```

Comment:
I'd add the entire command like `kusk api generate`
example: https://cloud.google.com/sdk/gcloud/reference/apigee